test/controllers/beneficial.owner.type.controller.spec.ts

```diff
@@ -71,10 +71,10 @@ const mockcheckEntityRequiresTrusts = checkEntityRequiresTrusts as jest.Mock;
 
 const mockGetTrustLandingUrl = getTrustLandingUrl as jest.Mock;
 
-const NEXT_PAGE_URL = "/NEXT_PAGE";
+const MOCKED_URL = "MOCKED_URL";
```

Comment:
Just wondering why this has been changed, should we keep it consistent with the other tests written for the redis changes?

Reply:
That value is returned by the mockGetUrlWithParamsToPath, so some of the new tests are using that value to build a url 'with params', but used in the middle of the url eg,
expect(text).toContain(Register_Prefix + NEXT_PAGE_URL + BO_IINDIVIDUAL_PAGE)

So rather than just have it exclusively for Next Pages, it made sense to have it as a generic 'mocked' url that can be used for next pages, middle of urls etc...
Some of the other tests have also already changed to MOCKED_URL